m5_calls.py

```diff
@@ -190,14 +194,24 @@ def traverse_ports(sym_catalog, symobject, simobject):
 def traverse_hierarchy_root(sym_catalog, symroot):
     """Recursively set parameters and then recursively set ports for
         instantiated objs"""
-    root = symroot.sim_object_instance
-    name, simroot = traverse_params(sym_catalog, symroot, root)
-    name, simroot = traverse_ports(sym_catalog, symroot, simroot)
+    try:
+        root = symroot.sim_object_instance
+        name, simroot = traverse_params(sym_catalog, symroot, root)
+        name, simroot = traverse_ports(sym_catalog, symroot, simroot)
+    except:
+        logging.error("Could not create simobject tree")
```

Comment:
You probably want to include the exception in the log. This way you will know *why* the simobject tree couldn't be created :)

Also, you should probably re-raise the exception into the GUI logic so you can pop up a window to tell the user that something went wrong.